machine/quanta/quanta_common_rglbmc/busybox/patches/onie-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,609 @@
+Add support for onie-syseeprom and mb_detect
```

Comment:
If this patch is *identical* to the patch in quanta_common_rangeley, then you can place a single copy of the patch in the top level "quanta/busybox" directory.  Then both machines can use the same patch.  The same can be done for kernel patches.

For example see the Accton boards.  The common patches are here:
https://github.com/opencomputeproject/onie/tree/master/machine/accton/busybox

which used by all the x86 Accton machines.

If that is helpful, please consider using that mechanism.